archive/tar_windows.go

```diff
@@ -351,11 +343,57 @@ func writeBackupStreamFromTarFile(w io.Writer, t *tar.Reader, hdr *tar.Header) (
 			Value: data,
 		})
 	}
-	if len(eas) != 0 {
-		eadata, err := winio.EncodeExtendedAttributes(eas)
+	eadata, err := winio.EncodeExtendedAttributes(eas)
+	if err != nil {
+		return nil, err
+	}
+	return eadata, nil
+}
+
+// If the given file is a symlink reads the ReparsePoint structure
+// from the tar header and encodes it into a byte array. If the file
+// is not a symlink then returns an empty byte array.
+func encodeReparsePointFromTarHeader(hdr *tar.Header) []byte {
+	if hdr.Typeflag != tar.TypeSymlink {
+		return []byte{}
+	}
+	_, isMountPoint := hdr.PAXRecords[hdrMountPoint]
+	rp := winio.ReparsePoint{
+		Target:       filepath.FromSlash(hdr.Linkname),
+		IsMountPoint: isMountPoint,
+	}
+	return winio.EncodeReparsePoint(&rp)
+}
+
+// writeBackupStreamFromTarFile writes a Win32 backup stream from the current tar file. Since this function may process multiple
+// tar file entries in order to collect all the alternate data streams for the file, it returns the next
+// tar file that was not processed, or io.EOF is there are no more.
+func writeBackupStreamFromTarFile(w io.Writer, t *tar.Reader, hdr *tar.Header) (*tar.Header, error) {
+	bw := winio.NewBackupStreamWriter(w)
+	var err error
```

Comment:
Why do we need this declared?

Reply:
Probably just leftover from the refactoring. Fixed.